docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,99 @@
+# Troubleshooting
```

Comment:
We should cross reference this with each steps readme.
Something like `Please refer to troubleshooting if you run into issues during this step` within prerequisites should gice the users a heads up. https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/1-org#prerequisites

Reply:
Done